src/core/stdc/math.d

```diff
@@ -874,6 +941,274 @@ extern (D)
+
+                   int    ilogb (double x);
+                   int    ilogbf(float  x);
+    extern(D)      int    ilogbl(real   x) { return ilogb(cast(double) x); }
+
+                   double ldexp (double x, int exp);
+                   float  ldexpf(float  x, int exp) { return cast(float) ldexp(x, exp); }
+    extern(D)      real   ldexpl(real   x, int exp) { return ldexp(cast(double) x, exp); }
+
+                   double log (double x);
+    version(Win64) float  logf(float  x);
+    version(Win32) float  logf(float  x) { return cast(float) log(x); }
+    extern(D)      real   logl(real   x) { return log(cast(double) x); }
+
+                   double log10 (double x);
+    version(Win64) float  log10f(float  x);
+    version(Win32) float  log10f(float  x) { return cast(float) log10(x); }
+    extern(D)      real   log10l(real   x) { return log10(cast(double) x); }
+
+                   double log1p (double x);
+                   float  log1pf(float  x);
+    extern(D)      real   log1pl(real   x) { return log1p(cast(double) x); }
+
+                   double log2 (double x);
+                   float  log2f(float  x);
+    extern(D)      real   log2l(real   x) { return log2(cast(double) x); }
+
+                   double logb (double x);
+                   float  logbf(float  x);
+    extern(D)      real   logbl(real   x) { return logb(cast(double) x); }
+
+                   double modf (double value, double* iptr);
+    version(Win64) float  modff(float  value, float*  iptr);
+    version(Win32) float  modff(float  value, float*  iptr)
+                   {
+                       double i;
+                       double r = modf(value, &i);
+                       *iptr = cast(float) i;
+                       return cast(float) r;
+                   }
+    extern(D)      real   modfl(real   value, real*   iptr)
+                   {
+                       double i;
+                       double r = modf(cast(double) value, &i);
+                       *iptr = i;
+                       return r;
+                   }
+
+                   double scalbn (double x, int n);
+                   float  scalbnf(float  x, int n);
+    extern(D)      real   scalbnl(real   x, int n) { return scalbn(cast(double) x, n); }
+
+                   double scalbln (double x, c_long n);
+                   float  scalblnf(float  x, c_long n);
+    extern(D)      real   scalblnl(real   x, c_long n) { return scalbln(cast(double) x, n); }
+
+                   double cbrt (double x);
+                   float  cbrtf(float  x);
+    extern(D)      real   cbrtl(real   x) { return cbrt(cast(double) x); }
+
+                   double fabs (double x);
+                   float  fabsf(float  x) { return cast(float) fabs(x); }
+    extern(D)      real   fabsl(real   x) { return fabs(cast(double) x); }
+
+    private        float  _hypotf(float x, float  y);
+                   double hypot (double x, double y);
```

Comment:
You should map the double version of hypot to _hypot, too. Otherwise linking against the DLL import library msvcrt.lib fails.